catalog/core/catalog-core-standardframework/src/main/java/ddf/catalog/event/retrievestatus/DownloadsStatusEventListener.java

```diff
@@ -29,14 +29,22 @@
 
   private static final Logger LOGGER = LoggerFactory.getLogger(DownloadsStatusEventListener.class);
 
+  private static final String ENTERING_STR = "ENTERING: {}";
```

Comment:
`catalog/core/catalog-core-impl/pubsub/src/main/java/ddf/catalog/pubsub/criteria/geospatial/GeospatialEvaluator.java`

Should the constant name be `ENTERING_MSG` or `ENTERING_STR`? (Same with `EXITING_...`)
Above file has it set to `ENTERING_MSG`, maybe that needs to be changed to `ENTERING_STR` or vice versa.